caddy/php-cli.go

```diff
@@ -0,0 +1,37 @@
+package caddy
+
+import (
+	"errors"
+	"os"
+
+	caddycmd "github.com/caddyserver/caddy/v2/cmd"
+	"github.com/dunglas/frankenphp"
+
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	caddycmd.RegisterCommand(caddycmd.Command{
+		Name:  "php-cli",
+		Usage: "script.php [args ...]",
+		Short: "Runs a PHP command",
+		Long: `
+Executes a PHP script similarly to the CLI SAPI.`,
+		CobraFunc: func(cmd *cobra.Command) {
+			cmd.DisableFlagParsing = true
+			cmd.RunE = caddycmd.WrapCommandFuncForCobra(cmdPHPCLI)
+		},
+	})
+}
+
+func cmdPHPCLI(fs caddycmd.Flags) (int, error) {
+	args := os.Args[2:]
+	if len(args) < 1 {
+		return 1, errors.New("the path to the PHP script is required")
+	}
```

Comment:
Makes me think, is there value in making `php-cli -i` work for example? (Could be useful to check enabled features). Some people use `php -a` and `php -l` on occasion too. But 🤷‍♂️

Reply:
I think it would be great if `./frankenphp php <args>` behaved identical to `php <args>` with php-cli installed.

Reply:
That would be nice, but it's a bit more work. The best option would be to allow the embedding of the official PHP SAPI (this requires a patch on PHP). I suggest merging this patch as is and trying to improve it later.